apps/glusterfs/placer_arbiter.go

```diff
@@ -148,6 +160,11 @@ func (bp *ArbiterBrickPlacer) Replace(
 		wds.Insert(i, d)
 	}
 	aopts := newArbiterOpts(opts)
+	// this is a mildly hacky way to deal with the higher level replace
```

Comment:
I assume a non-hacky way is not feasible right now? ;-)

Reply:
Yup